layers/+lang/python/README.org

```diff
@@ -174,8 +175,22 @@ Test commands start with ~m t~:
 | ~SPC m V~   | activate a virtual environment with  [[https://github.com/jorgenschaefer/pyvenv][pyvenv]]                                  |
 * Configuration
 ** Fill column
-If you want to customize the fill column value, use something like this inside the ~user-init~ function in your ~.spacemacs~:
+If you want to customize the fill column value, set the ~python-fill-column~ variable in the python layer config section:
 
 #+BEGIN_SRC elisp
-(setq python-fill-column 99)
+(setq-default dotspacemacs-configuration-layers '(
+    (python :variables python-fill-column 99)))
 #+END_SRC
```

Comment:
Okay, adding back the blank line. Is that a convention? Because I'm not seeing this in other README.org files.